src/main/java/nextstep/courses/domain/SessionEnrolment.java

```diff
@@ -4,67 +4,86 @@
 
 public class SessionEnrolment {
 
-    private SessionStudent sessionStudent;
-    private SessionStatusType sessionStatusType;
+    private SessionStudents sessionStudents;
+    private SessionStatus sessionStatus;
     private Amount amount;
     private boolean isFree;
```

Comment:
```suggestion
    private Amount amount;
    private boolean isFree;
```

`isFree`에 대한 상태가 어디로 가면 더 좋을까 고민하다가 하나의 다른 관점을 공유해봅니다!

[2. Don’t use the “else” keyword](https://developerhandbook.stakater.com/content/architecture/object-calisthenics.html#_2-don-t-use-the-else-keyword)

해당 문서에서 보면, 
`Early Return(빠른 리턴)`, `Solution with polymorphism(다형성)`개념에 대한 설명 보았을때,
`FreeAmount` 같은 객체로 도출해서 해결 할 수도 있을것 같네요! 
(지금도 충분히 좋은 구조입니다. 다양한 관점을 위해 공유해봅니다~ 😄 )